code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -204,7 +204,7 @@
 
 /obj/item/reagent_containers/food/drinks/holyoil
 	name = "flask of zelus oil"
-	desc = "A brass flask of Zelus oil, a viscous fluid with a scenting of brass - this flask may be sipped or thrown."
+	desc = "A brass flask of Zelus oil, a viscous fluid with a scenting of brass. Can be thrown to deal damage at a far."
```

Comment:
Personal suggestion, but you can change "at a far" to either "at a distance" or "from afar"